modules/variables.tf

```diff
@@ -189,7 +189,7 @@ variable "cognito_identity_providers" {
 
 variable "cognito_roles_attribute_admin_name" {
   type    = string
-  default = ""
+  default = "Admin"
```

Comment:
Not sure how this was behaving before without a default value, but I'm thinking we need this here.
(this is how we map cognito `group` attributes to app-level roles.)